Some Editorial changes proposed by Chris for final check (#34)
cheneyzhoucheng/network-digital-twin: draft-irtf-nmrg-network-digital-twin-arch.md

NOMS 2022 - TNT workshop {{TNT2022}}.

Although the application of digital twin technology in networking has
started, the research on digital twins for networks technology is
still in its infancy.  Current applications focus on specific
scenarios (such as network optimization), where a Network Digital Twin
is used as a network simulation tool to solve particular problems in
network operation and maintenance.  Combined with the characteristics
of digital twin technology and its application in other industries,
this document believes that Network Digital Twin can be regarded as
an indispensable part of the overall network system, and can play an
important role generally in architectures serving use cases across the
whole life cycle of a "real" (typically, physical) network. Such use cases

> and applications span the range of network operations such as network

Reviewer: ```suggestion
and applications span the range of network operations (e.g., network
```

Author: Good change

planning, construction, maintenance and optimization, and improve the

> automation and intelligence level of the network.

Reviewer: ```suggestion
automation and intelligence level of the network).
```

Author: Med, Can you check again whether this right 
parentheses is needed？

The role and utility of NDT may be represented architecturally by following
similar principles, e.g., {{ETSI-GS-ZSM-015}} or {{?RFC8969}}. As described in Section 7, an
NDT instantiation encompasses models, data, mapping, and interfaces. These
components then work in composition with other logic, functions or services
to deliver an overall functional architecture matching specific NDT use cases.

For example: an NDT instance may be used as a core element of an intent-drive network controller.
In such a case, an "outer" closed-loop (or, intent-assurance closed-loop) would detect
gaps between target service objectives set by intents and actual observed service characteristics,

> find proposed candidate mitigation solutions to soften the observed deviation, and drive the

Reviewer: ```suggestion
propose candidate mitigation solutions to soften the observed deviation, and drive the
```

Author: Good change
